src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -470,7 +473,10 @@ function unbound_add_host_entries($ifconfig_details = null)
             if (isset($config['dhcpd'][$interface]['enable']) && !empty($config['dhcpd'][$interface]['domain'])) {
                 $domain = $config['dhcpd'][$interface]['domain'];
             }
-            $unbound_entries .= "local-data-ptr: \"{$laddr} {$config['system']['hostname']}.{$domain}\"\n";
+            if ($interface === $interfaces[0] && !in_array($laddr, $ptr_records, true)) {
+                $unbound_entries .= "local-data-ptr: \"{$laddr} {$config['system']['hostname']}.{$domain}\"\n";
+                $ptr_records[] = $laddr;
+            }
             $unbound_entries .= "local-data: \"{$config['system']['hostname']}.{$domain} A {$laddr}\"\n";
```

Comment:
Maybe we should widen the conditional to guard A and AAAA as well?

Reply:
We should, but I'm not entirely sure if we can rely on the sorted list of interfaces and pick the first one. Shouldn't we at the very least determine if the picked interface is used for e.g. the webgui?

Reply:
So I've made e76f15c357a4645f for you to make this easier, maybe even move it out of the interfaces loop.
I tend to agree, but the metrics for the web GUI are difficult (what interfaces does the user want to listen on and what sort of NAT is he doing to change it to something else). So what about trying to aim for the same logic as anti-lockout would?  That's a little further but not too complicated.

Reply:
@fichtner https://github.com/opnsense/core/pull/5925/commits/e5633b81de3bbb7ee8c00ce1b841446b3e18cbb5. What do you think?